index.js

```diff
@@ -55,7 +55,7 @@ Merry.prototype.router = function (opts, routes) {
       // val should ideally be a stream already, but if it's not we got you bae
       handler(req, res, ctx, function (err, val) {
         if (err) {
-          res.statusCode = err.statusCode || res.statusCode || 500
+          res.statusCode = err.statusCode || 500
```

Comment:
I think sometimes we might want to set an explicit status code, how about something like this:
```js
res.statusCode = (err.statusCode >= 400) ? err.statusCode : 500
```